lib/paymentPage/components/defaultBackButton.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/material.dart';
+
+class DefaultBackButton extends StatelessWidget {
```

Comment:
@batsy3 this can be moved to the widgets folder. For every component appearing on each page kindly move it to the widgets as a reusable.